src/core/src/main/java/org/apache/jmeter/gui/GuiPackage.java

```diff
@@ -91,12 +97,6 @@
      */
     private boolean dirty = false;
 
-    /**
-     * Map from TestElement to JMeterGUIComponent, mapping the nodes in the tree
-     * to their corresponding GUI components.
-     */
-    private Map<TestElement, JMeterGUIComponent> nodesToGui = new HashMap<>();
```

Comment:
In case you wonder: this `map` was not really useful. It duplicated the contents from `guis` and `testBeanGUIs`